contracts/bonding/BondingManager.sol

```diff
@@ -1553,6 +1606,10 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         return IRoundsManager(controller.getContract(keccak256("RoundsManager")));
     }
 
+    function treasury() internal view returns (address payable) {
+        return payable(controller.getContract(keccak256("Treasury")));
```

Comment:
Might be beyond my technical depth on what the payable function means here, but in spirit the treasury can receive any asset, including ETH. Even if the Livepeer protocol only will send it LPT.

Reply:
The payable keyword here doesn't impact the Treasury contract's ability to receive ETH and is just used to inform the compiler and the programmer that the address will need be able to receive ETH from code implemented in this contract (BondingManager). So, if the BondingManager never needs to send ETH to the Treasury technically speaking the payable keyword isn't necessary.

Reply:
I think I ended up adding the `payable` when I was converting this to the explicit type `Treasury/TimelockController`, which has payable functions and thus the compiler complains when you try to convert a non-payable address to it. Then I stopped referencing the specific contract implementation here and just returned an address and didn't realize that I didn't need the payable conversion anymore. Will fix!

Reply:
This was actually a leftover from a previous version where I returned the concrete instance of the contract. Since the `Treasury` (`TimelockController`) has a fallback payable function, solidity requires the address to be converted to `payable` first.

As we're only returning the address now this is not necessary anymore, so I've removed the `payable` conversion. Thanks for noticing this.

Reply:
lol github was buggy for me yesterday, I could swear I had already replied here but it wasn't showing me